### PR TITLE
Hide DAO token input for DXdao

### PR DESCRIPTION
**Pull request by nicoelzer**
- We want to hide the input "DAO token (DXD) Reward" for the DXDao as this token is not used and a duplicate of the actual DXD token.

dxdao

### CODE REVIEW

**Comment by orenyodfat** [comment]
please fix lint issue @nicoelzer

**Comment by nicoelzer** [comment]
@orenyodfat updated version just passed the lint tests.

File: src/components/Proposal/Create/SchemeForms/CreateContributionRewardProposal.tsx [hunk 1]
```diff
@@ -375,7 +375,7 @@ class CreateContributionReward extends React.Component<IProps, IStateProps> {
                   </div>
                 </div>
 
-                <div className={css.reward}>
+                <div className={daoAvatarAddress === "0x519b70055af55a007110b4ff99b0ea33071c720a" ? css.hidden : css.reward}>
```
Comment (dkent600) [inline src/components/Proposal/Create/SchemeForms/CreateContributionRewardProposal.tsx:378]: Ouch this is ugly

In any case, DAO addresses should not be assumed unique across chains.  @orenyodfat

**Comment by dkent600** [comment]
@orenyodfat If this token is a duplicate, why have we not simply removed or renamed the duplicate?

**Pull request merged**